xml/deployment_cluster_administration.xml

```diff
@@ -92,55 +94,56 @@
      To add a new user, create a LDIF file like this:
     </para>
     <screen>
-     <![CDATA[
-dn: uid=<userid>,ou=People,dc=infra,dc=caasp,dc=local
+dn: uid=<replaceable>userid</replaceable>,ou=People,dc=infra,dc=caasp,dc=local
 objectClass: person
 objectClass: inetOrgPerson
 objectClass: top
-uid: <userid>
-userPassword: <password hash>
-givenname: <first name>
-cn: <full name>
-sn: <surname>
-mail: <email address>
-     ]]>
+uid: <replaceable>userid</replaceable>
+userPassword: <replaceable>password hash</replaceable>
+givenname: <replaceable>first name</replaceable>
+cn: <replaceable>full name</replaceable>
+sn: <replaceable>surname</replaceable>
+mail: <replaceable>email address</replaceable>
     </screen>
     <para>
-     To generate the password hash, install the <package>openldap2</package>
-     package:
-    </para>
-    <screen>&prompt.root; <command>zypper</command> <option>install openldap2</option></screen>
-    <para>
-     Then generate the password hash:
+     To generate the password hash, on a separate machine, install the 
+     <package>openldap2</package> package:
     </para>
+    <screen>
+&prompt.root; <command>zypper</command> <option>install openldap2</option>
+    </screen>
+    <para>Then generate the password hash: </para>
     <screen>&prompt.root;<command>/usr/sbin/slappasswd</command></screen>
     <para>
-     Make sure to replace all the parameters in the template above, indicated
-     as &lt;....&gt;. The mail attribute value is used as the login to
+     Make sure to replace all the parameters indicated <replaceable>like this</replaceable>
+     in the template above. The mail attribute value is used as the login to
      &dashboard; and &kube;.
     </para>
     <para>
-     Populate your OpenLDAP server with this LDIF file:
+     Transfer the resulting LDIF file to your &productname; server and populate your 
```

Comment:
For the record: You don't transfer the file, you use the LDAP client tools to connect to the server.